gobblin-iceberg/src/main/java/org/apache/gobblin/iceberg/writer/GobblinMCEWriter.java

```diff
@@ -420,4 +434,22 @@ public static State setHiveRegProperties(State state, GobblinMetadataChangeEvent
     }
     return tmpState;
   }
+
+  /**
+   * Submit event indicating that a specific set of GMCEs have been skipped, so there is a gap in the registration
+   */
+  private void submitFailureEvent(GobblinMetadataException exception) {
+    log.warn(String.format("Sending GTE to indicate table flush failure for %s", exception.tableName));
+
+    GobblinEventBuilder gobblinTrackingEvent = new GobblinEventBuilder(IcebergMCEMetadataKeys.ICEBERG_COMMIT_EVENT_NAME);
```

Comment:
Do we want to use the same event name when we successfully commit one snapshot? Should we use another event name like METADATA_WRITER_FAILURE_EVENT?

Reply:
Whoops yeah I meant to change that but forgot